### PR TITLE
Get TOC Working to split the huge JSON

### PR DESCRIPTION
**Pull request by SalocinDotTEN**
Worked with Jeff during TechUp April 2021 and got Stoplight to finally read off the table of contents JSON.
Split up of the huge file can begin.

### CODE REVIEW

**Review by jeff-knurek — APPROVED** [review]
I wonder if the file rename:
`.stoplight.json → old.stoplight.json`
is even needed now? maybe better to just delete that json file completely....

**Comment by SalocinDotTEN** [comment]
> I wonder if the file rename:
> `.stoplight.json → old.stoplight.json`
> is even needed now? maybe better to just delete that json file completely....

Probably yes.
Stoplight giving issues again. Unable to view the branch. Branch selector only in the studio and not in the documentation view.

File: docs/index.md [hunk 1]
```diff
@@ -1,3 +1,75 @@
 ## Introduction.
 
 This documents the Application Programming Interface (API) for the Project Lockdown ecosystem.
+
+They serve
+- Back Office (BO)
+- Mapping Platform (MAP)
+- Public Open Data
+
```
Comment (jeff-knurek) [inline docs/index.md:9]:
```suggestion
## Icon definition:
```

**Review by jeff-knurek — COMMENTED** [review]
I see the branch select in the ["documentation" view ](https://theiofoundation.stoplight.io/docs/projectlockdown/branches/branch-with-toc/docs/index.md) :thinking: 

Screenshot from 2021-04-21 11-23-07

**Comment by SalocinDotTEN** [comment]
> I see the branch select in the ["documentation" view ](https://theiofoundation.stoplight.io/docs/projectlockdown/branches/branch-with-toc/docs/index.md) 🤔
> 
> Screenshot from 2021-04-21 11-23-07

Weird. It did not appear.

**Comment by jeff-knurek** [comment]
looks like there's a conflict in files (`API/Reference/Project-Lockdown.v1.json` :shrug: ). maybe you need to merge master into your branch and fix that.

**Review by SalocinDotTEN — COMMENTED** [review]
Updated the json file with master.

**Comment by SalocinDotTEN** [comment]
great! I was wondering where the branch went.
Stoplight does not show the "update from master" and those little branch buttons underneath.
I suppose it is now automated.
So my own branch is now updated with the TOC file. :)

**Pull request merged**